tests/virtual_machines_test.go

```diff
@@ -627,6 +598,26 @@ func newNetwork(name string) kubevirtv1.Network {
 	}
 }
 
+// This function allocates vms with 1 NIC each, in order to fill the mac pool to the brim.
+func AllocateVmsUntilFull(macsToLeaveFree uint64) error {
+	maxPoolSize := getMacPoolSize()
+	Expect(maxPoolSize > macsToLeaveFree).To(BeTrue(), "max pool size must be greater than the number of macs we want to leave free")
```

Comment:
Use gomega `BeNumerically(">",` as a rule of thumb use the most specific gomega matcher always so you get proper expected/obtained messages.